features/daily/call_record.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Call Record
+
+  Scenario: Default call with extension callrecord enabled
+    Given call record directories are empty
```

Comment:
I disagree with this step: removing all data is not acceptable for a precondition: we did that in the first tests that were written and we regretted it quickly:
* one of the reasons is that you can't run this test on an instance that is used for other things than running the acceptance tests. For example, it would erase all recordings on my development instance.
* another reason is that if we ever install a recording file as a precondition asset file (e.g. to test resilience to weird filenames), then this test would remove the asset file.

However, removing all recordings of a specific user created specifically for the test is acceptable.

Reply:
fixed